ui/webui/src/installation/InstallationLanguage.jsx

```diff
@@ -162,19 +160,15 @@ export const InstallationLanguage = ({ onSelectLang }) => {
     };
 
     return (
-        <PageSection>
-            <Form>
-                <Title headingLevel="h2" size="1xl">
-                    WELCOME TO FEDORA...
-                </Title>
-                <FormGroup label={_("What language would you like to use during the installation process?")}>
-                    <LanguageSelector lang={lang} onSelectLang={setLang} />
-                </FormGroup>
-                <ActionGroup>
-                    <Button id="continue-btn" variant="primary" onClick={handleOnContinue}>{_("Continue")}</Button>
-                    <Button variant="link">{_("Quit")}</Button>
-                </ActionGroup>
-            </Form>
-        </PageSection>
+        <Form>
+            <Title headingLevel="h2" size="1xl">
+                WELCOME TO FEDORA...
+            </Title>
+            <FormGroup label={_("What language would you like to use during the installation process?")}>
+                <LanguageSelector lang={lang} onSelectLang={setLang} menuAppendTo={document.body}/>
+            </FormGroup>
+            <ActionGroup>
```

Comment:
You can delete the ActionGroup since you don't use it now.

Reply:
Removed. :)